tests/test_database.py

```diff
@@ -0,0 +1,39 @@
+
+import unittest
+from app.dojo import *
+
+
+class TestDatabase(unittest.TestCase):
+    def setUp(self):
+        self.dojo = Dojo()
+
+    def test_successfully_saves_state(self):
+        """ Test saves application state to Database. """
+        self.dojo.create_room('living', ["kigali"])
+        self.dojo.create_room('office', ["Django"])
+        self.dojo.add_person("Maina", "oscar", "fellow")
+        output = self.dojo.save_state("flask.db")
+
+        #connect to db and create session
+        engine = create_engine('sqlite:///database/somedb.db')
```

Comment:
It's imported in the dojo file, on this file i am importing everything from dojo